Calculator/src/Calculator.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String operator;
+        double result;
+
+        System.out.println("First value:");
+        double firstvalue = CorrectNumb();
+
+        System.out.println("Console operation:");
+        operator = sc.next();
+
+        System.out.println("Second value:");
+        double secondvalue = CorrectNumb();
```

Comment:
please move variable initialization in one place (like where result)

Reply:
I have corrected